java-showcase/gapic-showcase/pom.xml

```diff
@@ -281,7 +281,7 @@
         <dependency>
           <groupId>ch.qos.logback</groupId>
           <artifactId>logback-classic</artifactId>
-          <version>1.3.15</version>
+          <version>1.2.13</version>
```

Comment:
Restore the version to fix compilation error.

Reply:
curious what was the compilation error? I did not came across when changing to 1.3.x [here](https://github.com/googleapis/sdk-platform-java/pull/3686). 
Also, can you keep `logback-classic` and `logback-core` versions consistent. Perhaps add a property for them

Reply:
The compilation error is https://github.com/googleapis/sdk-platform-java/pull/3698#issuecomment-2722682530.

> I did not came across when changing to 1.3.x [here](https://github.com/googleapis/sdk-platform-java/pull/3686).

I think the showcase test got removed before merging #3686.

Reply:
hmm. interesting, I might look into this error later. 
For now, I agree lets use 1.2.x versions for both